### PR TITLE
Add SQL Server page

### PR DESCRIPTION
**Pull request by scsmithr**

### CODE REVIEW

File: docs/data-sources/supported/sql-server.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+<!-- prettier-ignore-start -->
+# SQL Server
+{: .no_toc }
+<!-- prettier-ignore-end -->
+
+<!-- markdownlint-enable MD022 -->
+
+SQL Server is able to be used as an external data source. Either an entire
+database or a single table may be added as a data source.
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:18]:
```suggestion
GlareDB supports SQL Server as an external data source. You can configure both single 
tables and entire databases from SQL Server.
```

File: docs/data-sources/supported/sql-server.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+<!-- prettier-ignore-start -->
+
+- TOC
+{:toc}
+<!-- prettier-ignore-end -->
+
+## Connect a SQL Server database
+
+An entire SQL Server database can be added using the [CREATE EXTERNAL DATABASE]
+command.
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:29]:
```suggestion
Add an entire SQL Server database with the [CREATE EXTERNAL DATABASE]
command.
```

File: docs/data-sources/supported/sql-server.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+{: .important}
+
+> There are two equivalent formats. In both formats, `database-name` will be the
+> name of the database inside GlareDB. This cannot be qualified, and must be
+> unique across all other databases in the deployment.
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:35]:
```suggestion
> There are two equivalent forms: In both, `<database-name>` will be the
> name of the database in GlareDB. This cannot be qualified, and must be
> unique across all other databases in the deployment.
```

File: docs/data-sources/supported/sql-server.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+### Database options
+
+| Field               | Description                                                    |
+| ------------------- | -------------------------------------------------------------- |
+| `connection_string` | An [ADO connection string] pointing to the SQL Server instance |
+
+## Connect a single table
+
+Adding an external table can be done through the [CREATE EXTERNAL TABLE]
+command.
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:56]:
```suggestion
Add an external table with the [CREATE EXTERNAL TABLE]
command.
```

File: docs/data-sources/supported/sql-server.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+{: .important}
+
+> There are two equivalent formats. In both formats, `table-name` will be the
+> name of the database inside GlareDB. `table-name` may optionally be qualified
+> with a schema name.
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:62]:
```suggestion
> There are two equivalent forms: in both forms, `table-name` becomes the
> name of the database inside GlareDB. `table-name` may be qualified
> with a schema name.
```

File: docs/data-sources/supported/sql-server.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+ );
+```
+
+### Database options
+
+| Field               | Description                                                    |
+| ------------------- | -------------------------------------------------------------- |
+| `connection_string` | An [ADO connection string] pointing to the SQL Server instance |
+
+## Connect a single table
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:53]:
```suggestion
## Add a Single Table
```

File: docs/data-sources/supported/sql-server.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+```sql
+CREATE EXTERNAL DATABASE <database-name>
+ FROM sql_server
+ OPTIONS (
+   connection_string = '<connection_string>'
+ );
+```
+
+### Database options
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:47]:
```suggestion
### Database Options
```

File: docs/data-sources/supported/sql-server.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+An entire SQL Server database can be added using the [CREATE EXTERNAL DATABASE]
+command.
+
+{: .important}
+
+> There are two equivalent formats. In both formats, `database-name` will be the
+> name of the database inside GlareDB. This cannot be qualified, and must be
+> unique across all other databases in the deployment.
+
+### Database format
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:37]:
```suggestion
### Database Format
```

File: docs/data-sources/supported/sql-server.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+SQL Server is able to be used as an external data source. Either an entire
+database or a single table may be added as a data source.
+
+<!-- prettier-ignore-start -->
+
+- TOC
+{:toc}
+<!-- prettier-ignore-end -->
+
+## Connect a SQL Server database
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:26]:
```suggestion
## Add a SQL Server Database
```

File: docs/data-sources/supported/sql-server.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+CREATE EXTERNAL TABLE <table-name>
+ FROM sql_server
+ OPTIONS (
+  connection_string = '<connection_string>',
+  schema = '<schema>',
+  table = '<table>'
+);
+```
+
+### Table options
```
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md:76]:
```suggestion
### Table Options
```

**Review by tychoish — APPROVED** [review]
you mentioned two different forms, but I only see one? 

style nits:
- headings should be in title case
- stronger verb phrases.

File: docs/data-sources/supported/sql-server.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+<!-- prettier-ignore-start -->
+# SQL Server
+{: .no_toc }
+<!-- prettier-ignore-end -->
+
+<!-- markdownlint-enable MD022 -->
+
+GlareDB supports SQL Server as an external data source. You can configure both single 
+tables and entire databases from SQL Server.
```
Comment (greyscaled) [inline docs/data-sources/supported/sql-server.md:18]: Personally, I'd remove the word `"both"` here

**Review by greyscaled — APPROVED** [review]

File: docs/data-sources/supported/sql-server.md (hunk 11 above)
Comment (tychoish) [inline docs/data-sources/supported/sql-server.md]:
```suggestion
GlareDB supports SQL Server as an external data source. You can configure single 
tables and entire databases from SQL Server.
```

**Comment by greyscaled** [comment]
@tychoish Created a `next` branch - updating PR target

**Pull request merged**